storm-client/src/jvm/org/apache/storm/daemon/worker/WorkerState.java

```diff
@@ -88,6 +88,7 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(WorkerState.class);
     private static final long LOAD_REFRESH_INTERVAL_MS = 5000L;
+    private static final int RESEND_BACKPRESSURE_SIZE = 10000;
```

Comment:
Am wondering if it makes sense to make this configurable?

Reply:
I can make this change if someone more authoritative than myself concurs that it is a good idea.

Reply:
I don't have a strong opinion, just wanted to know what you thought. If you don't see a need, we can always do it later if someone feels it is necessary.

Reply:
Sounds good. Thanks for considering it.